upload-blocks/BlockComponent/BlockComponent.js

```diff
@@ -225,11 +239,6 @@ export class BlockComponent extends BaseComponent {
     return this.$['*uploadCollection'];
   }
 
-  /** @param {String} shortKey */
-  cfg(shortKey) {
-    return this.getCssData('--cfg-' + shortKey, true);
```

Comment:
maybe we won't remove it? the new way to access the config properties looks weird (`this.$['*--cfg-prop']`)